src/rgw/store/dbstore/dbstore_mgr.h

```diff
@@ -33,15 +33,6 @@ class DBStoreManager {
     cct = _cct;
 	default_db = createDB(default_tenant);
   };
-  DBStoreManager(string logfile, int loglevel): DBStoreHandles() {
```

Comment:
This is currently used at "https://github.com/ceph/ceph/blob/master/src/rgw/store/dbstore/dbstore_main.cc#L148" . That test file may soon be obsolete. So I am okay if it is removed.

Reply:
oops, i missed that! i added the constructor back, but replaced `dout_subsys` with `ceph_subsys_rgw`

that constructor *should* go away, because the CephContext should be managed outside of the store. and note that `global_init()` returns a reference-counted `boost::intrusive_ptr<CephContext>`, but this class only stores a raw pointer to that - which means the intrusive_ptr would go out of scope and delete the `CephContext` right after `global_init()` returns

Reply:
ah okay. Thanks! I will address it along with the other cleanup you have mentioned in the tracker.